plugins/TodayAtMun/diary.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
This data might not need to live in the repo, since eventually it will be outdated.
could you instead make it so when the bot starts, and `def __init__(self, manifest, bot):` in the `__init__.py` method is called, it pulls down and generates this data instead?

Reply:
Will work on this now.

Reply:
Done